interface/billing/sl_eob_search.php

```diff
@@ -52,9 +52,11 @@
 	
 	function is_auth_portal( $pid = 0){
 		if ($pData = sqlQuery("SELECT * FROM `patient_data` WHERE `pid` = ?", array($pid) )) {
-			if($pData['allow_patient_portal'] != "YES")
-				return false;
-				else return true;
+			if($pData['allow_patient_portal'] != "YES") return false;
+			else{
+				$_SESSION['portalUser'] = strtolower($pData['fname']) . $pData['id'];
+				return true;
+			}
```

Comment:
We are trying to move to PSR-2 standards, so rec following this standard structure for this if/else:
http://www.php-fig.org/psr/psr-2/#if-elseif-else